cfy_manager/components/postgresql_server/postgresql_server.py

```diff
@@ -582,10 +586,25 @@ def _configure_cluster(self):
                     logger.info('Configuring etcd authentication')
                     # Note that, per the etcd documentation, the root user
                     # must exist if authentication is to be enabled for etcd
-                    self._etcd_command(
+                    user_create_result = self._etcd_command(
                         ['user', 'add', 'root'],
                         stdin=cluster_config['etcd']['root_password'],
+                        ignore_failures=True,
                     )
+
+                    if user_create_result.returncode != 0:
+                        stdout = user_create_result.aggr_stdout.strip()
+                        stderr = user_create_result.aggr_stderr.strip()
+                        exists = 'User root already exists'
+                        if exists in stderr or exists in stdout:
```

Comment:
surely we know which one is it going to be, stdout or stderr?

Reply:
Different approach being used now, so closing this thread.